server/chainquery/queries/claimQueries.js

```diff
@@ -68,15 +68,35 @@ export default (db, table, sequelize) => ({
     });
   },
 
-  getAllChannelClaims: async (channelClaimId, bidState) => {
+  getAllChannelClaims: async (channelClaimId, params) => {
     logger.debug(`claim.getAllChannelClaims for ${channelClaimId}`);
-    const whereClause = bidState || {
```

Comment:
See here and the original signature of `getAllChannelClaims: async (channelClaimId, bidState)`, this allows all claims to be fetched.  The changes break this pattern and force Spee.ch filtering on code that should be unopinionated.

Reply:
I had read that once you go beyond 1 optional parameter, you're better off passing an object. With bidState already being optional and not passed from the controller, contentTypes would have to be a second optional parameter, meaning the function, I think, would have to be called with ( first, null, third ). Is there a better way to make contentTypes optional?  I'm leading toward a GetMatchingChannelClaims function in chainquery instead. This would pave the way for future db queries.